README.md

```diff
@@ -124,8 +133,8 @@ serve .next
 
 To connect to your own Meilisearch instance, two environment variables should be provided:
 
-- `HOST`: the URL of your Meilisearch instance
-- `API_KEY`: an API key with at least search rights
+- `HOST`: the URL of your Meilisearch instance.
+- `API_KEY`: an API key with at least search rights.
```

Comment:
Can you update this with `MEILISEARCH_HOST` and `MEILISEARCH_API_KEY` ? 
You can also remove the dots at the end 😉

Reply:
I had added the full stop to commit changes in the readme😂 cool i will make the changes.

Reply:
I have done the changes 

> Can you update this with `MEILISEARCH_HOST` and `MEILISEARCH_API_KEY` ? You can also remove the dots at the end 😉

I have made the changes!